src/Book.js

```diff
@@ -61,13 +79,35 @@ export const Book = () => {
         ) : (
           <h1>LOADING BOOK</h1>
         )}
-        {pages.map(({id, page_image, page_description}) => (
+        {pages.map(({id, page_image, page_description, page_number}) => (
           <div key={id}>
-            <img src={page_image} height="300px" alt="poop" />
+            <img
+              src={page_image}
+              height="300px"
+              alt={'Page number: ' + page_number}
+            />
             <p>{page_description}</p>
+            <button
+              className="btn btn-danger btn-sm"
+              onClick={deletePage(page_number)}
```

Comment:
We should be editing and deleting by something that is unique and can never change: the page id.